int8_t spd0
int8_t spd1
int8_t spd2
int8_t spd3